app/main/src/main.ts

```diff
@@ -73,7 +73,7 @@ function createMainWindow() {
 	win = new BrowserWindow({
 		width: 340,
 		height: getFrameHeight(),
-		resizable: true,
+		resizable: false,
```

Comment:
in ubuntu it break  fullscreen so after set it true it work again
so please check it on ubuntu

Reply:
Weirdly setting this to true breaks it on mac. Or at least i believe it was this property. So I have a task this weekend to look into it with all 3 systems open at once to find one that doesn't break something :(